network-config-analyzer/TcpLikeProperties.py

```diff
@@ -275,11 +274,20 @@ def print_diff(self, other, self_name, other_name):
         other_minus_self = other - self
         diff_str = self_name if self_minus_other else other_name
         if self_minus_other:
-            item = self_minus_other.get_first_item()
-            diff_str += f' allows communication on {item} while {other_name} does not [source-destination pair] '
+            diff_str += f' allows communication on {self_minus_other._get_first_item_str()} while {other_name} does not'
```

Comment:
Don't we have expected-output tests that needed to be modified?

Reply:
No, because the output for `print_diff` is relevant for equivalence query, and the output-tests are currently only for connectivity map and semantic diff queries.

Reply:
Right! I confused the two.